projects/microphysics/argo/prog-run-and-eval.yaml

```
@@ -0,0 +1,163 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: prognostic-microphysics-
+spec:
+  entrypoint: prognostic-run
+  arguments:
+    parameters:
+      - name: tag
+      - name: config
+      - name: tf_model
+      - name: baseline_tag
+        value: "baseline"
+      - name: on_off_flag
+        value: "--online"
+      - name: do_piggy
+        value: "true"
+  volumes:
+  - name: gcp-key-secret
+    secret:
+      secretName: gcp-key
+  templates:
+  - name: prognostic-run
+    steps:
+    - - name: run-model
+        template: run-model
+        arguments:
+          parameters:
+          - name: tag
+            value: "{{workflow.parameters.tag}}"
+          - name: config
+            value:  "{{workflow.parameters.config}}"
+          - name: tf_model
+            value: "{{workflow.parameters.tf_model}}"
+          - name: on_off_flag
+            value: "{{workflow.parameters.on_off_flag}}"
+    - - name: piggyback-diags
+        template: piggyback-diags
+        when: "{{workflow.parameters.do_piggy}} == true"
+        arguments:
+          parameters:
+          - name: tag
+            value: "{{workflow.parameters.tag}}"
+      - name: prognostic-diags
+        template: prognostic-diags
+        when: "'{{workflow.parameters.on_off_flag}}' == '--online'"
+        arguments:
+          parameters:
+          - name: tag
+            value: "{{workflow.parameters.tag}}"
+          - name: baseline_tag
+            value: "{{workflow.parameters.baseline_tag}}"
+  - name: run-model
+    inputs:
+      parameters:
+      - name: tag
+      - name: config
+      - name: tf_model
+      - name: on_off_flag
+    tolerations:
+    - key: "dedicated"
+      operator: "Equal"
+      value: "med-sim-pool"
+      effect: "NoSchedule"
+    container:
+      image: "us.gcr.io/vcm-ml/prognostic_run:811d4a88b8b8f29d727f2d61f46f96b770c8ab47"
+      imagePullPolicy: Always
+      workingDir: "/fv3net/projects/microphysics"
+      resources:
+        requests:
+          memory: "6Gi"
+          cpu: "7500m"
+        limits:
+          memory: "8Gi"
+          cpu: "7500m"
+      envFrom:
+      - secretRef:
+          name: wandb-andrep
+      env:
+      - name: GOOGLE_APPLICATION_CREDENTIALS 
+        value: /secret/gcp-credentials/key.json
```

Move `wandb_run_group` here.

```suggestion
        value: /secret/gcp-credentials/key.json
      - name: WANDB_RUN_GROUP
        value: "{{inputs.parameter.tag}}"
```